fixtures/cameo/ts-100-ww.json

```diff
@@ -59,40 +50,40 @@
         {
           "dmxRange": [6, 255],
           "type": "StrobeSpeed",
```

Comment:
This should be a ShutterStrobe capability.

Reply:
Done